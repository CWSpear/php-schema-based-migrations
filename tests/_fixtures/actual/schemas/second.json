{
    "fields": {
        "id": {
            "type": "integer",
            "limit": 10,
            "identity": true,
            "unsigned": "unsigned"
        },
        "id_two": {
            "type": "integer",
            "limit": 10,
            "unsigned": "unsigned"
        },
        "id_three": {
            "type": "integer",
            "limit": 10,
            "unsigned": "unsigned"
        },
        "id_four": {
            "type": "integer",
            "limit": 10,
            "unsigned": "unsigned"
        },
        "id_five": {
            "type": "integer",
            "limit": 10,
            "unsigned": "unsigned"
        },
        "id_six": {
            "type": "integer",
            "limit": 10,
            "unsigned": "unsigned"
        },
        "id_seven": {
            "type": "integer",
            "limit": 10,
            "unsigned": "unsigned"
        }
    },
    "foreignKeys": [],
    "indexes": [
        {
            "columns": [
                "id"
            ],
            "unique": true
        },
        {
            "columns": [
                "id_two"
            ],
            "unique": true
        },
        {
            "columns": [
                "id_three",
                "id_four"
            ],
            "unique": true
        },
        {
            "columns": [
                "id_five"
            ],
            "unique": false
        },
        {
            "columns": [
                "id_six",
                "id_seven"
            ],
            "unique": false
        }
    ]
}