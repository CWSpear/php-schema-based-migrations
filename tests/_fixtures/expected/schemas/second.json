{
    "fields": {
        "id": {
            "type": "integer",
            "identity": true,
            "unsigned": true
        },
        "id_two": {
            "type": "integer",
            "unsigned": true
        },
        "id_three": {
            "type": "integer",
            "unsigned": true
        },
        "id_four": {
            "type": "integer",
            "unsigned": true
        },
        "id_five": {
            "type": "integer",
            "unsigned": true
        },
        "id_six": {
            "type": "integer",
            "unsigned": true
        },
        "id_seven": {
            "type": "integer",
            "unsigned": true
        }
    },
    "foreignKeys": [],
    "indexes": {
        "primary": {
            "columns": [
                "id"
            ],
            "primary": true,
            "unique": true
        },
        "id_two": {
            "columns": [
                "id_two"
            ],
            "unique": true
        },
        "id_three_id_four": {
            "columns": [
                "id_three",
                "id_four"
            ],
            "unique": true
        },
        "id_five": {
            "columns": [
                "id_five"
            ]
        },
        "id_six_id_seven": {
            "columns": [
                "id_six",
                "id_seven"
            ]
        }
    }
}